### DISCUSSION TITLE
Is any UAD release from a specific vendor a candidate for this project?

### DISCUSSION THREAD

**Discussion by HayesE** [pal1000/Realtek-UAD-generic#129] [Q&A] [answered]
This isn't a passive aggressive nag for the absolute latest version to be included here, but just wondering what is the general criteria for a UAD release to be included in this generic UAD project. I noticed ASRock in particular are more or less the forefront of UAD version releases lately but it doesn't trigger much action here or other driver mods until there's an Asus or MSI UAD subsequently.

**Comment by pal1000** [comment, marked as answer]
I usually look on [MDL](https://forums.mydigitallife.net/threads/update-realtek-high-definition-audio.72236/) and if I find a newer version I create a generic package.  If same newer version from multiple vendors is available I select the vendor with newest software components. If still multiple vendors match I pick at random.
